Userland/Utilities/groups.cpp

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2021, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/Vector.h>
+#include <LibCore/Account.h>
+#include <LibCore/ArgsParser.h>
+#include <grp.h>
+#include <unistd.h>
+
+static void print_account_gids(const Core::Account& account)
+{
+    auto* gr = getgrgid(account.gid());
+    if (!gr) {
+        outln();
+        return;
+    }
+
+    out("{}", gr->gr_name);
+    for (auto& gid : account.extra_gids()) {
+        gr = getgrgid(gid);
+        out(" {}", gr->gr_name);
```

Comment:
If we need to do a nullptr check above, do we also need one here?

Reply:
The nullptr check at line 7 is due to user like `nona` not having a primary group ; `account.extra_gids()` seems to correctly return an empty array in this particular case. I don't think it is needed, but we can always add one for symetry? :sweat_smile:

Reply:
That's not true, every user has a primary group (GID). For `nona` that's 200 as you can see in `/etc/passwd`. It might just not be present in `/etc/group`.
However, now that I think about it I believe this is is fine, as the extra GIDs should always match the entries in `/etc/group`, which `getgrgid()` (indirectly) reads from.

Reply:
sorry I meant `nona` not having its group in `/etc/group`: 
![image](https://user-images.githubusercontent.com/61476054/117157567-fa51fa00-adbe-11eb-8f4a-522868140758.png)